test/unit/spec/v2/view-builder/views/CaptchaView_spec.js

```diff
@@ -160,4 +172,114 @@ describe('v2/view-builder/views/CaptchaView', function() {
 
     window.grecaptcha = original;
   });
+
+  describe('ALTCHA Captcha', function() {
+    it('view renders correctly with ALTCHA configuration', function() {
+      testContext.init(captchaAltchaObject.captcha.value);
+      expect(testContext.view.el).toMatchSnapshot('with ALTCHA configuration');
+    });
+
+    it('ALTCHA Captcha gets loaded with correct URL', function() {
+      const spy = jest.spyOn(CaptchaView.prototype, '_loadCaptchaLib');
+      testContext.init(captchaAltchaObject.captcha.value);
+      expect(spy).toHaveBeenCalledWith('https://cdn.jsdelivr.net/gh/altcha-org/altcha/dist/altcha.min.js');
+    });
+
+    it('ALTCHA uses altcha-captcha class in template', function() {
+      testContext.init(captchaAltchaObject.captcha.value);
+      expect(testContext.view.$el.find('#captcha-container').hasClass('altcha-captcha')).toBe(true);
+    });
+
+    it('ALTCHA callback creates altcha-widget element with correct attributes', function() {
+      testContext.init(captchaAltchaObject.captcha.value);
+      
+      // Simulate the onAltchaCaptchaLoaded callback
+      window.OktaSignInWidgetOnCaptchaLoaded();
+      
+      const altchaWidget = testContext.view.$el.find('altcha-widget')[0];
+      expect(altchaWidget).toBeDefined();
+      expect(altchaWidget.getAttribute('floating')).toBe('true');
+      expect(altchaWidget.getAttribute('hidefooter')).toBe('true');
+      expect(altchaWidget.getAttribute('hidelogo')).toBe('true');
+      expect(altchaWidget.getAttribute('challengeurl')).toBe('http://localhost:3000/api/v1/altcha');
+    });
+
+    it('ALTCHA callback sets up temp token and triggers onCaptchaLoaded', function() {
+      testContext.init(captchaAltchaObject.captcha.value);
+      const spy = jest.spyOn(testContext.view.options.appState, 'trigger');
+      
+      window.OktaSignInWidgetOnCaptchaLoaded();
+      
+      expect(testContext.view.model.get('captchaVerify.captchaToken')).toEqual('tempToken');
+      expect(testContext.view.$el.find('#captcha-container').attr('data-captcha-id')).toEqual('altcha');
+      expect(spy).toHaveBeenCalledWith('onCaptchaLoaded', expect.any(HTMLElement));
+    });
+
+    it('ALTCHA widget strings attribute contains localized strings', function() {
+      testContext.init(captchaAltchaObject.captcha.value);
+      
+      window.OktaSignInWidgetOnCaptchaLoaded();
+      
+      const altchaWidget = testContext.view.$el.find('altcha-widget')[0];
```

Comment:
nit: you checked `expect(altchaWidget).not.toBeNull()` in other tests. I think it's appropriate here as well following a `.find()` call